.pre-commit-config.yaml

```diff
@@ -25,26 +25,16 @@ repos:
     rev: 22.12.0
     hooks:
       - id: black
-  - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.189
     hooks:
-      - id: flake8
-        exclude: (^ibis/tests/sql/snapshots/|_py310\.py$)
+      - id: ruff
+        # makes sure to exclude a file even if it's passed in explicitly
+        args: ["--force-exclude"]
   - repo: https://github.com/pycqa/docformatter
     rev: v1.5.1
     hooks:
       - id: docformatter
-  - repo: https://github.com/MarcoGorelli/absolufy-imports
```

Comment:
Is ruff handling it?

Reply:
It's handling checking for new relative imports using the [`"TID"`](https://github.com/charliermarsh/ruff#flake8-tidy-imports-tid) code.